src/plugins/legacy.js

```diff
@@ -38,7 +38,7 @@ export type LegacyCommand = {
 }
 
 export function convertFromV5 (c: LegacyCommand) {
-  if (!c.topic) throw new Error('command has no topic')
+  if (c.topic == null) throw new Error('command has no topic')
```

Comment:
The requirement on our end is to support mapping a command to only a namespace, in which case the topic and command args to the modified functions here are `''`.

I'll link the source example to you in Slack.